src/renderer/features/operations/OperationSubmit/model/submit-model.ts

```diff
@@ -35,7 +35,7 @@ const formSubmitted = createEvent<ExtrinsicResultParams>();
 const extrinsicSucceeded = createEvent<ExtrinsicResultParams>();
 const extrinsicFailed = createEvent<string>();
 
-const $submitStore = restore<Input>(formInitiated, null);
+const $submitStore = restore<Input>(formInitiated, null).reset(formSubmitted);
```

Comment:
I've seen we have it for other common stores. Do we need it here as well in this case?

Reply:
I guess this won't hurt anyone, 👍